plugins/woocommerce/package.json

```diff
@@ -786,8 +786,10 @@
 				"node_modules/@woocommerce/e2e-core-tests/CHANGELOG.md",
 				"node_modules/@woocommerce/api/dist/",
 				"node_modules/@woocommerce/admin-e2e-tests/build",
+				"node_modules/@woocommerce/classic-assets/build",
```

Comment:
Left these in, because they're generated for every PR now so I think they should be on trunk